huntsman/tests/test_camera.py

```diff
@@ -311,7 +313,7 @@ def test_autofocus_coarse(camera, patterns):
         pytest.skip("Camera does not have a focuser")
     try:
         autofocus_event = camera.autofocus(coarse=True)
-        autofocus_event.wait()
+        autofocus_event.wait(timeout=30)
```

Comment:
This timeout is way too short. This is the cause of the cryptic 'not enough data received' errors. The simulated cameras are perhaps a bit too realistic in that a simulated autofocus sequence actually takes quite a long time. Changing this to 150 seconds makes this test run just fine.